api/app/auth.py

```diff
@@ -76,3 +76,8 @@ async def auth_with_set_fire_starts_role_required(token=Depends(authentication_r
 async def auth_with_select_station_role_required(token=Depends(authentication_required)):
     """ Only return requests that have set fire starts permission """
     return await check_token_for_role('hfi_select_station', token)
+
+
+async def auth_with_admin_role_required(token=Depends(authentication_required)):
```

Comment:
```suggestion
async def auth_with_station_admin_role_required(token=Depends(authentication_required)):
```
Maybe more explicit?